src/class/Tab/Tab.cpp

```diff
@@ -0,0 +1,85 @@
+
+#include <Tab/Tab.hpp>
+
+/*
+** ------------------------------- CONSTRUCTOR --------------------------------
+*/
+
+Tab::Tab()
+{
+}
+
+/*
+** -------------------------------- DESTRUCTOR --------------------------------
+*/
+
+Tab::~Tab()
+{
+}
+
+/*
+** --------------------------------- OVERLOAD ---------------------------------
+*/
+
+std::ostream &operator<<(std::ostream &o, Tab const &i)
+{
+	char **tab = i.toCTable();
+	o << "Tab[";
+	for (int j = 0; tab[j] != NULL; j++)
+	{
+		o << tab[j] << std::endl;
+	}
+	o << "]";
+	return o;
+}
+
+/*
+** --------------------------------- METHODS ----------------------------------
+*/
+
+char **Tab::toCTable(void) const
+{
+	char **tab = (char **)calloc(this->_tab.size() + 1, sizeof(char *));
```

Comment:
i don't see we can use calloc()
and also we can't use libft...
we need to ask/think of how to create a c table